ownCloudAppShared/Client/File Lists/ClientQueryViewController.swift

```diff
@@ -136,6 +254,20 @@ open class ClientQueryViewController: QueryFileListTableViewController, UIDropIn
 
 	private var viewControllerVisible : Bool = false
 
+	open override func viewDidAppear(_ animated: Bool) {
+ 		super.viewDidAppear(animated)
+
+ 		searchController?.delegate = self
+ 	}
+//
+//	open override func startQuery(_ theQuery: OCQuery) {
+//		if theQuery == customSearchQuery {
+//			self.updateCustomSearchQuery()
+//		} else {
+//			super.startQuery(theQuery)
+//		}
+//	}
+
```

Comment:
Is this comment needed or can it be removed?

Reply:
Thanks, that's a leftover from a previous refactoring. Removed it with the latest commits.